src/image/root_image.cpp

```diff
@@ -0,0 +1,77 @@
+#include "root_image.hpp"
+
+namespace lucidy{
+   
+    RootImage::RootImage(){
```

Comment:
settings should be initialized using constructor initializers

Reply:
Do not agree, does note make a difference in performance of readability

Reply:
It does make a difference in performance. Since the `init()` function is virtual. Which means it has to be looked up at runtime what address the function is at. This consumes both flash memory for the V-Table and instructions when calling this function for dereferencing the function pointer. Placing the content of `init()` removes the necessity for a virtual function. 
It can be said that placing the content of init inside the constructor makes the code less maintainable, but that only really applies when you have a large `init()` function and more than one constructor calling `init()`
You can decide what to do, but I cannot assure that the development-to-master review will allow it.